frontend/src/containers/PlayerPage/PlayerPage.js

```diff
@@ -197,22 +226,40 @@ class PlayerPage extends React.Component {
                 title={video.title}
                 showArrow
                 onIconsClick={[
-                  toggleVideoInformation,
+                  this.toggleInformationDrawer,
                   handleVideoDownload,
                   this.toggleDeletionDialog,
                 ]}
                 onActionIconClick={this.handleArrowBackClick}
                 iconsToShow={[InfoIcon, DownloadIcon, DeleteIcon]}
               />
             </div>
+            <InformationDrawer
+              open={showInformationDrawer}
+              onOpen={this.toggleInformationDrawer}
+              onClose={this.toggleInformationDrawer}
+              videoTitle={video.title}
+              videoDuration={video.duration}
+              videoSize={video.size}
+              videoFormat={video.format}
+              videoResolution={video.resolution}
```

Comment:
You don't really have to change anything here if you don't want to, but just passing a whole `video` object would reduce the number of lines here

Reply:
I thought this was a bit more declarative making this more of a dummy component, but sure